### PR TITLE
Split terms.html

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
[Set up Hotjar](https://trello.com/c/KLHW48Fc/18-set-up-hotjar)

File: cove/templates/terms.html [hunk 1]
```diff
@@ -1,217 +1,30 @@
 {% extends 'base.html' %}
 {% load i18n %}
 {% block content %}
+    {% with app=request.current_app %}
+        <div>
+            {# The content is split into many small templates.
+               This is so if one cove instance wants to change only one small bit of the page, they can do so by providing replacement templates of only the bit they want to change.
+               For example, a cove instance may want to keep most of the page the same but add legal content for another tracker.
+            #}
```
Comment (Bjwebb) [inline cove/templates/terms.html]: Note these comment tags don't work in Django, instead use https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#comment

Comment (ghost) [reply cove/templates/terms.html]: Don't they? I did test it, wierd .... ok.

Comment (ghost) [reply cove/templates/terms.html]: fixed and tested again

**Review by Bjwebb — APPROVED** [review]

File: cove/templates/terms.html (hunk 1 above)
Comment (Bjwebb) [reply cove/templates/terms.html]: Weird. Are you testing with a later Django version?

**Pull request merged**